scout/server/blueprints/institutes/templates/overview/causatives.html

```diff
@@ -59,7 +59,7 @@
               </a>
             </td>
             <td><span class="text-muted" style="display: inline-block; margin-left: 10px;" data-toggle='tooltip' data-container='body' title="{{ group_variant.reference|truncate(30, True) }} → {{ group_variant.alternative|truncate(30, True) }}">
-                {{ group_variant.reference|truncate(10, True) }} → {{ group_variant.alternative|truncate(10, True) }}
+                {{ group_variant.reference|truncate(10, True) }} → {{ group_variant.alternative|truncate(10, True) | replace("<", " ") | replace(">", "")}}
```

Comment:
The ideal is to loose the ref and arrow for the strs, so something like (🚧 untested code ahead 🚧)
```suggestion
                {% if group_variant.category in 'str' %}
                    {{ group_variant.alternative|truncate(10, True) | replace("<", " ") | replace(">", "")}}
                {% else %}
                    {{ group_variant.reference|truncate(10, True) }} → {{ group_variant.alternative|truncate(10, True) }}
                {% endif%}                
```
and the same for the hover and variant.html?